datalad/distribution/get.py

```diff
@@ -668,7 +670,13 @@ def __call__(
         refds = require_dataset(
             dataset, check_installed=True, purpose='get content')
 
+        path = assure_list(path)
+        # resolve input paths against dataset in order to compare absolute paths in results
+        # to the given paths later on
+        # TODO: Not yet clear whether to use dataset or refds here. Why is dataset used in the call to Subdatasets?
```

Comment:
It always has to be `dataset` never a dataset instance, because path arguments always have to be evaluated in the context of the given dataset (if any), not a discovered dataset.

Reply:
Right. That makes sense.